.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,6 @@
 - id: docformatter-conda
   name: docformatter-conda
-  entry: docformatter -i
+  entry: docformatter -i --config pyproject.toml
```

Comment:
This is in contrast to what the upstream hook does: https://github.com/PyCQA/docformatter/blob/1def3c413021cdc0d355eb988444fbc847181173/.pre-commit-hooks.yaml#L4

We try to follow as closely as possible what the non-conda variant does. Can you pass that instead in your pre-commit-config via `args:`?

Reply:
@xhochy I tried this before and, to my surprise, I got the following error messages:

```
[Errno 2] No such file or directory: '--config pyproject.toml'
[Errno 2] No such file or directory: '--config pyproject.toml'
[Errno 2] No such file or directory: '--config pyproject.toml'
[Errno 2] No such file or directory: '--config pyproject.toml'
[Errno 2] No such file or directory: '--config pyproject.toml'
```

Hook configuration:

```yaml
- repo: https://github.com/Quantco/pre-commit-mirrors-docformatter
  rev: 1.5.0
  hooks:
    - id: docformatter-conda
      args:
        - --config pyproject.toml
```

It seems like `docformatter` assumes that the arguments are files (?). If I use two distinct args for `--config` and `pyproject.toml`, it doesn't throw an error but it doesn't read the config either. Do you think that we can fix this by applying the following change to our conda hook?

```yaml
entry: docformatter
args: ["-i"]
```

Reply:
Your YAML syntax is incorrect. It should be:

```
args:
  - --config
  - pyproject.toml
```

Reply:
@xhochy I also tried that, that's what I meant with the following 😄 

> If I use two distinct args for --config and pyproject.toml...

As pointed out above, this doesn't cause any errors being printed but still does not respect the configuration in `pyproject.toml`.

Reply:
On a different note, typing `docformatter -i` in the shell _does_ respect the `pyproject.toml` file without providing it as an explicit option 🧐

Reply:
Can you link me to an repo/commit where I can test this behaviour?

Reply:
@xhochy I tried it with https://github.com/Quantco/sql-compyre (branch `add-docformatter`).